.chromatic/custom-addons/chromatic/index.js

```diff
@@ -25,41 +27,78 @@ export const withChromaticProvider = makeDecorator({
       height = options.height;
     }
 
-    let expressTheme = colorSchemes.length === 1 ? expressThemes[colorSchemes[0]] : expressThemes.light;
-    let expressColorScheme = colorSchemes.length === 1 ? colorSchemes[0].replace(/est$/, '') : 'light';
-    let expressScale = scalesToRender.length === 1 ? scalesToRender[0] : 'medium';
-    let expressLocale = selectedLocales.length === 1 ? selectedLocales[0] : 'en-US';
-
-    // do not add a top level provider, each provider variant needs to be independent so that we don't have RTL/LTR styles that interfere with each other
-    return (
-      <DisableAnimations disableAnimations={options.disableAnimations}>
-        <div style={{display: 'flex', flexDirection: 'column', height, minHeight}}>
-            {colorSchemes.map(colorScheme =>
-              scalesToRender.map(scale =>
-                (colorScheme === 'light' ? selectedLocales : ['en-US']).map(locale =>
-                  <Provider key={`${colorScheme}_${scale}_${locale}`} theme={themes[colorScheme]} colorScheme={colorScheme.replace(/est$/, '')} scale={scale} locale={locale} typekitId="pbi5ojv">
-                    <View margin="size-100">
-                      <h1 style={{margin: 0, padding: 0}}>{`${colorScheme}, ${scale}, ${locale}`}</h1>
-                      {getStory(context)}
-                    </View>
-                  </Provider>
-                )
-              )
-            )}
-            {options.express !== false &&
-              <Provider key="express" theme={expressTheme} colorScheme={expressColorScheme} scale={expressScale} locale={expressLocale} typekitId="pbi5ojv">
-                <View margin="size-100">
-                  <h1 style={{margin: 0, padding: 0}}>express, {expressColorScheme}, {expressScale}, {expressLocale}</h1>
-                  {getStory(context)}
-                </View>
-              </Provider>
-            }
-        </div>
-      </DisableAnimations>
-    )
+    if (context.title.includes('S2/')) {
+      return <RenderS2 getStory={getStory} context={context} options={options} selectedLocales={selectedLocales} height={height} minHeight={minHeight} />
+    } else {
+      return <RenderV3 getStory={getStory} context={context} options={options} selectedLocales={selectedLocales} height={height} minHeight={minHeight} />
+    }
   }
 });
 
+function RenderS2({getStory, context, options, selectedLocales, height, minHeight}) {
+  let colorSchemes = options.colorSchemes || S2ColorThemes;
+  // TODO: not sure if we need to snapshot against layers? Doesn't look like any of the colors of the components change with respect to the layer
+  let backgrounds = options.backgrounds || S2Backgrounds;
```

Comment:
will need to check the designs, but I did note some components like https://www.chromatic.com/test?appId=5f0dd5ad2b5fc10022a2e320&id=66d2011fb8fc882bbab275c8 that seem to lack contrast in some of the layers

Reply:
avatar border color responds to the layer color
just look for the `--background-s2-color` or whatever that custom property is called. I think it's in Cards too.

Reply:
probably few enough that we can just do it for those components specifically rather than making noisy sets of snapshots with things that don't change

Reply:
sweet, thanks for tip with the `--background-s2-color`, I'll do an audit